api/v2.0/swagger.yaml

```diff
@@ -9016,3 +9144,89 @@ definitions:
         type: string
         format: date-time
         description: The creation time of the accessory
+
+  ScanDataExportRequest:
+    type: object
+    description: The criteria to select the scan data to export.
+    properties:
+      job_name:
+        type: string
+        description: Name of the scan data export job
+      projects:
+        type: array
+        items:
+          type: integer
+          format: int64
+        description: A list of one or more projects for which to export the scan data, defaults to all if empty
+      labels:
+        type: array
+        items:
+          type: integer
+          format: int64
+        description: A list of one or more labels for which to export the scan data, defaults to all if empty
+      repositories:
+        type: string
+        description: A list of repositories for which to export the scan data, defaults to all if empty
+      cveIds:
+        type: string
+        description: CVE-IDs for which to export data. Multiple CVE-IDs can be specified by separating using ',' and enclosed between '{}'. Defaults to all if empty
+      tags:
```

Comment:
do we need the tags here? Since the scan result is associated with the digest, rather than tag. Repository is enough, IMO.